### PR TITLE
RN 39739

### PR DESCRIPTION
**Pull request by EdithLansens**
@GillesBara, could you please review this pull request to confirm the accurate inclusion of release note 39739 in the user guide? Thanks in advance!

### CODE REVIEW

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Customize_Component_Layout.md [hunk 1]
```diff
@@ -28,7 +28,17 @@ Each component in a dashboard or low-code app has a number of default options. B
 
      - In the *Colors* section, specify a custom background color and/or font color, either by specifying the color in RGB format or by using the color picker box on the right.
 
-       From DataMiner 10.0.12 onwards, under *Colors* > *Color palette*, you can customize additional component colors, e.g. for the lines in a line chart.
+       Under *Colors* > *Data colors* or *Colors* > *Color palette* (prior to DataMiner 10.3.0 [CU16]/10.4.0 [CU4]/10.4.7<!--RN 39739-->), you can customize additional component colors, e.g. for the lines in a line chart.
```
Comment (GillesBara) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Customize_Component_Layout.md:31]: Just a headsup, this structure changes again in https://intranet.skyline.be/DataMiner/Lists/Release%20Notes/DispForm2.aspx?ID=39746: 
image

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Customize_Component_Layout.md [hunk 2]
```diff
@@ -28,7 +28,17 @@ Each component in a dashboard or low-code app has a number of default options. B
...
+
+       From DataMiner 10.3.0 [CU16]/10.4.0 [CU4]/10.4.7 onwards<!--RN 39739-->, under *Colors* > *Data colors*, you can specify conditional colors. If a display label of the data matches the specified text, that data will inherit the conditional color.
+
+       To add a conditional color:
+
+       1. Select *Add conditional color*.
+
+       1. Enter the text to match the display label. Partial matches are allowed. For example, entering `Tot` will match data labeled `Total`.
```
Comment (GillesBara) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Customize_Component_Layout.md:39]: It's actually a regular expression. '^Tot$' will only match 'Tot'.

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Customize_Component_Layout.md [hunk 3]
```diff
@@ -28,7 +28,17 @@ Each component in a dashboard or low-code app has a number of default options. B
...
+
+       1. Choose a custom color, either by specifying the color in RGB format or by using the color picker box on the right.
```
Comment (GillesBara) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Customize_Component_Layout.md:41]: RGB isn't the only accepted format, you can also use the hex or names: https://www.w3schools.com/tags/ref_colornames.asp

**Review by GillesBara — CHANGES_REQUESTED** [review]

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Customize_Component_Layout.md (hunk 2 above)
Comment (EdithLansens) [reply user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Customize_Component_Layout.md:39]:
```suggestion
       1. Enter a regular expression to match the display label. For example, entering `Tot` will match data labeled `Total`, while entering `^Tot$` will only match data labeled `Tot`.
```
@GillesBara, is this more accurate?

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Customize_Component_Layout.md (hunk 3 above)
Comment (EdithLansens) [reply user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Customize_Component_Layout.md:41]:
```suggestion
       1. Choose a custom color, either by specifying the color in RGB format, by entering the hex value or HTML color name, or by using the color picker box on the right.
```

**Review by GillesBara — APPROVED** [review]

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**